Makefile

```diff
@@ -452,6 +452,14 @@ sam-deploy-package: guard-artifact_bucket guard-artifact_bucket_prefix guard-sta
 		--force-upload \
 		--tags "version=$$VERSION_NUMBER" \
 		--parameter-overrides \
+			TruststoreVersion=$$LATEST_TRUSTSTORE_VERSION \
+			TruststoreFile=$$TRUSTSTORE_FILE \
 			VersionNumber=$$VERSION_NUMBER \
 			CommitId=$$COMMIT_ID \
-			Env=$$TARGET_ENVIRONMENT
+			Env=$$TARGET_ENVIRONMENT \
+			DomainNameExport=$$DOMAIN_NAME_EXPORT \
+			ZoneIDExport=$$ZONE_ID_EXPORT \
+			VPC=$$VPC \
```

Comment:
These arnt actually defined in the main template and thus arnt actually used, but as per other comments these dont need to be passed in at this level and can just be directly imported in the template.

Reply:
Redundant values have been removed.